kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/LogReplayEngineMetricsSuite.scala

```diff
@@ -445,6 +445,32 @@ class LogReplayEngineMetricsSuite extends AnyFunSuite with TestUtils {
     }
   }
 
+  test("checksum not found at the read version, but found at a previous version; " +
+    "crc exists after version queried") {
```

Comment:
nit: this test case mentions both `checksum` and `crc` -- can we just pick one terminology?